makefile.shade

```diff
@@ -9,3 +12,40 @@ custom-goals
 
 #initialize if='Directory.Exists("src")'
   exec program='dotnet' commandline='run -p ../../tools/Microsoft.AspNetCore.Server.Kestrel.GeneratedCode Internal/Http/FrameHeaders.Generated.cs Internal/Http/Frame.Generated.cs' workingdir='${kestrelSrcDir}'
+
+#initialize if='IsOSX()'
```

Comment:
There're are ways, but its probably less change to tack this on to the existing target.